backend/src/user/user.service.ts

```diff
@@ -70,4 +72,39 @@ export class UserService {
 
     return existingUser || undefined;
   }
+  public async getBookmarks(id: number): Promise<string[]> {
+    const existingUser = await this.userRepository.findOne({
+      id: id,
+    });
+
+    return existingUser.bookmarks || undefined;
```

Comment:
Wouldn't it make more sense to return an empty array here than `undefined`

Especially since the signature indicates it will return a `Promise<string[]>`

Reply:
yes